index.js

```diff
@@ -33,3 +33,11 @@ module.exports.getRequestHeaders = function () {
 module.exports.respondWith = function (data, options) {
     return window.fetch.returns(Promise.resolve(new Response(data, options)));
 };
+
+Object.defineProperty(
+    module.exports,
+    'called',
+    {
+        get: () => !!window.fetch.firstCall
```

Comment:
Allow function recommended by `helmut` linting, although my initial instinct was to use function syntax
```suggestion
        get: function() { return !!window.fetch.firstCall; }
```